### PR TITLE
1 class is added

### PR DESCRIPTION
**Pull request by Milky34**
Review my code

### CODE REVIEW

File: AshaPractice_github/src/main/java/Newpackage/Practice.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package Newpackage;
+
+public class Practice {
+
+	public static void main(String[] args) {
```
Comment (Milky34) [inline AshaPractice_github/src/main/java/Newpackage/Practice.java:5]: Review this code

**Review by Milky34 — COMMENTED** [review]
Tell me the Updates.....

**Comment by Asha306** [comment]
use x path 
don't use index

**Comment by Milky34** [comment]
Ok mam

**Comment by Asha306** [comment]
merged done

**Pull request merged**